prime-router/docker-compose.yml

```diff
@@ -140,20 +140,17 @@ services:
       - prime-router_build
 
   web_receiver:
-    image: nginx:latest
-    ports:
-      - "8090:80"
+    image: node:14-alpine
```

Comment:
What are this changes here?

Reply:
to make the frontend-react changes work. This was breaking with the removal of the `frontend` folder

Reply:
@whytheplatypus  you beat me to it :) [my-commit](https://github.com/CDCgov/prime-reportstream/pull/3062/commits/77f9a61a0e1eaeb3dd543d74ca82898fcafa26ad)

Will your changes fix the issue with running .cleanslate.sh? If so, I can delete my PR.

Reply:
@JosiahSiegel oh awesome! lets use yours so it's not tied to the rest of this CI work.

Reply:
Sure. Once your commit is merged into master, i'll fetch your changes and test on my branch. Up to you if you want to back out any commits.